Banner mask loading does not work with jar

```
Apr 12 08:55:39 obey java[1238]: Caused by: java.io.FileNotFoundException: class path resource [mask-96.png] cannot be resolved to absolute file path because it does not reside in the file system: jar:file:/var/www/backendtest/bannergress-backend.jar!/BOOT-INF/classes!/mask-96.png
Apr 12 08:55:39 obey java[1238]:         at org.springframework.util.ResourceUtils.getFile(ResourceUtils.java:217) ~[spring-core-5.3.5.jar!/:5.3.5]
Apr 12 08:55:39 obey java[1238]:         at org.springframework.core.io.AbstractFileResolvingResource.getFile(AbstractFileResolvingResource.java:154) ~[spring-core-5.3.5.jar!/:5.3.5]
Apr 12 08:55:39 obey java[1238]:         at com.bannergress.backend.services.impl.BannerPictureServiceImpl.<clinit>(BannerPictureServiceImpl.java:77) ~[classes!/:na]
```

--- a/src/main/java/com/bannergress/backend/services/impl/BannerPictureServiceImpl.java
+++ b/src/main/java/com/bannergress/backend/services/impl/BannerPictureServiceImpl.java
@@ -7,7 +7,6 @@ import com.bannergress.backend.services.BannerPictureService;
 import com.google.common.hash.Hasher;
 import com.google.common.hash.Hashing;
 import org.springframework.beans.factory.annotation.Autowired;
-import org.springframework.core.io.ClassPathResource;
 import org.springframework.stereotype.Service;
 import org.springframework.transaction.annotation.Isolation;
 import org.springframework.transaction.annotation.Transactional;
@@ -74,7 +73,7 @@ public class BannerPictureServiceImpl implements BannerPictureService {
 
     protected static final BufferedImage maskImage;
     static {
-        try (InputStream stream = new ClassPathResource("mask-96.png").getInputStream()) {
+        try (InputStream stream = BannerPictureServiceImpl.class.getResourceAsStream("/mask-96.png")) {
             maskImage = ImageIO.read(stream);
         } catch (IOException ex) {
             throw new RuntimeException(ex);

